lib/ReactFitText.js

```diff
@@ -48,8 +48,13 @@ module.exports = React.createClass({
                       parseFloat(this.props.minFontSize)) + 'px';
   },
   _renderChildren: function(){
-    return React.Children.map(this.props.children, child =>
-      React.cloneElement(child,{ref:(c) => this._childRef = c})
+    return React.Children.map(this.props.children, function(child) {
+      return React.cloneElement(child, {
+          ref: function(c) {
+            return this._childRef = c
+          }.bind(this)
+        })
+      }.bind(this)
```

Comment:
Sorry, the _renderChildren function should be replace by:

  _renderChildren: function _renderChildren() {
    var _this = this;

    return React.Children.map(this.props.children, function (child) {
      return React.cloneElement(child, { ref: function ref(c) {
        return _this._childRef = c;
      } });
    });
  },

I've tested it and it works.